introduction.md

```diff
@@ -4,4 +4,9 @@ I'm Julian.
 
 I don't have a favorite path yet.
 I'll explore then and maybe recommit this file in future.
-But so far is JavaScript.
\ No newline at end of file
+But so far is JavaScript.
+
+My deadly skills are:
+* Git
+* JavaScript
+* Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?